src/chains/ethereum/ethereum/tests/helpers/getProvider.ts

```diff
@@ -22,8 +23,12 @@ const getProvider = async (
 
   // don't write to stdout in tests
   if (!options.logging.logger) {
+    throw new Error("doesn't ever get here");
     options.logging.logger = { log: () => {} };
   }
+  if (!options.miner.instamine) {
+    options.miner.instamine = "greedy";
+  }
```

Comment:
Noting the default in `src/chains/ethereum/ethereum/src/api.ts` is `strict` but setting the default options for test to `greedy`

Reply:
My mistake.